testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/raw/RawITSupport.java

```diff
@@ -35,6 +36,7 @@
 /**
  * Support class for raw ITs.
  */
+@Category(RawTestGroup.class)
```

Comment:
As discussed with @cstamas this does appear to be working so I'm going to merge in.
As well, contiperf 2.4.3 doesn't appear to be published to central.